### PR TITLE
Support ROCm v4.3 in document

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

**Comment by kmaehashi** [comment]
This PRis blocked until all ROCm tests failures are resolved.

**Comment by kmaehashi** [comment]
Could you mention `LLVM_PATH` issue in `install.rst`?

File: docs/source/install.rst
Comment (kmaehashi) [inline docs/source/install.rst:393]:
```suggestion
   ROCm 4.3 is known to have an issue related to clang/HIP. If you experienced an error like ``fatal error: '__clang_hip_runtime_wrapper.h' file not found``, please try setting an environment variable ``LLVM_PATH``, e.g., ``export LLVM_PATH=/opt/rocm-4.3.0/llvm``. See `#5592 <https://github.com/cupy/cupy/issues/5592#issuecomment-901136342>`_ for details.


```

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Review by kmaehashi — APPROVED** [review]

**Comment by takagi** [comment]
The note is not rendered? https://cupy--5633.org.readthedocs.build/en/5633/install.html#using-cupy-on-amd-gpu-experimental

File: docs/source/install.rst
Comment (kmaehashi) [inline docs/source/install.rst:398]: Let's merge it inline
```suggestion
   ROCm 4.3 is known to have an issue related to clang/HIP. If you experienced an error like ``fatal error: '__clang_hip_runtime_wrapper.h' file not found``, please try setting an environment variable ``LLVM_PATH`` (e.g., `export LLVM_PATH="${ROCM_HOME}/llvm"`). See `#5592 <https://github.com/cupy/cupy/issues/5592#issuecomment-901136342>`_ for details.
```

**Comment by kmaehashi** [comment]
I'll merge after confirming RTD build

File: docs/source/install.rst
Comment (kmaehashi) [inline docs/source/install.rst:393]:
```suggestion
   ROCm 4.3 is known to have an issue related to clang/HIP. If you experienced an error like ``fatal error: '__clang_hip_runtime_wrapper.h' file not found``, please try setting an environment variable ``LLVM_PATH`` (e.g., `export LLVM_PATH="${ROCM_HOME}/llvm"`). See `#5592 <https://github.com/cupy/cupy/issues/5592#issuecomment-901136342>`_ for details.

```

**Pull request merged**
